openwisp_radius/apps.py

```diff
@@ -14,12 +16,27 @@
 from .utils import load_model, update_user_related_records
 
 
-class OpenwispRadiusConfig(AppConfig):
+class OpenwispRadiusConfig(ApiAppConfig):
     name = 'openwisp_radius'
     label = 'openwisp_radius'
     verbose_name = 'Freeradius'
 
+    API_ENABLED = True
+    REST_FRAMEWORK_SETTINGS = {
+        'DEFAULT_THROTTLE_RATES': {
+            # Strict throttling Policy
+            'throttle_obtain_auth': default_or_test('100/hour', None),
+            'throttle_validate_auth': default_or_test('100/hour', None),
+            # Lenient throttling Policy
+            'throttle_crea_ph_tok': default_or_test('250/hour', None),
+            'throttle_vali_ph_no': default_or_test('250/hour', None),
```

Comment:
`throttle_vali_ph_no` and `throttle_crea_ph_tok` aren't the most friendly names, is there a reason for choosing them over `throttle_valid_phone` and `throttle_create_phone_token`?

Reply:
No @atb00ker , There is no such reason I just tried to shorten the variable name, I'll rename it to `throttle_valid_phone`, `throttle_create_phone_token`, and so on... this will be more convenient I guess.

Reply:
yes please, use readable names

Reply:
Done renamed it.